README.md

```diff
@@ -1,91 +1,95 @@
+# EVM Placeholder proof system verifier 
 
 [![Discord](https://img.shields.io/discord/969303013749579846.svg?logo=discord&style=flat-square)](https://discord.gg/KmTAEjbmM3)
 [![Telegram](https://img.shields.io/badge/Telegram-2CA5E0?style=flat-square&logo=telegram&logoColor=dark)](https://t.me/nilfoundation)
 [![Twitter](https://img.shields.io/twitter/follow/nil_foundation)](https://twitter.com/nil_foundation)
 
-This repository contains the smart contracts for validating zero knowledge proofs 
-generated in placeholder proof system in EVM. 
+This repository contains the smart contracts for validating zero-knowledge proofs generated
+via [Placeholder proof system](https://github.com/NilFoundation/evm-placeholder-verification) in-EVM.
 
 ## Dependencies
 
 - [Hardhat](https://hardhat.org/)
-- [nodejs](https://nodejs.org/en/) >= 16.0
+- [nodejs](https://nodejs.org/en/) - Hardhat requires an LTS version; as of August 2023 it's v18.17.0
 
-
-## Clone
-```
+## Clone the project
+```bash
 git clone git@github.com:NilFoundation/evm-placeholder-verification.git
 cd evm-placeholder-verification
 ```
 
 ## Install dependency packages
-```
+```bash
 npm i
 ```
 
 ## Compile contracts
-```
+```bash
 npx hardhat compile
 ```
 
 ## Deploy
 
-Launch a local-network using the following
-```
+Launch a local network using the following command:
+```bash
 npx hardhat node
 ```
 
-To deploy to test environment (ex: Ganache)
-```
-npx hardhat deploy  --network localhost 
+To deploy to a test environment (Ganache, for example), run:
+```bash
+npx hardhat deploy --network localhost
 ```
 
-Hardhat re-uses old deployments, to force re-deploy add the `--reset` flag above
+Hardhat reuses old deployments by default; to force re-deploy, add the `--reset` flag to the command.
 
-## Verification of zkLLVM circtuit compiler output
+## Verification of zkLLVM circuit compiler output
 
-zkLLVM compiler prepares circuits in form of instantiated contracts, which have to be deployed to the blockchain.
+[zkLLVM compiler](https://github.com/NilFoundation/zkllvm) prepares circuits
+as instantiated contracts that can be deployed to a blockchain.
 
-Once you got zkLLVM output - put a circuit directory it under `contracts/zkllvm`. The folder should contain following files:
+Once you get zkLLVM output, place a circuit directory under `contracts/zkllvm`.
+That folder should contain the following files:
 
 ```
-* proof.bin -- placeholder proof file
-* circuit_params.json -- parameters file in JSON format
-* public_input.json -- public input file in JSON format
-* linked_libs_list.json -- list of external libraries, have to be deployed for gate argument computation.
-* gate_argument.sol, gate0.sol, ... gateN.sol -- solidity files with gate argument computation
+* proof.bin — Placeholder proof file
+* circuit_params.json — parameters file
+* public_input.json — public input file
+* linked_libs_list.json — list of external libraries that have to be deployed for gate argument computation
```

Comment:
Have to be deployed by whom?

Reply:
If I understand it correctly, linked_libs_list.json is required for automation of deploing. If you upload gate arguments to the blockhain manually, you don't need it